### DISCUSSION TITLE
[@types/lodash]: Missing signatures for includes method

### DISCUSSION THREAD

**Discussion by saksham-sharma** [DefinitelyTyped/DefinitelyTyped#69430] [Issues with a @types package] [answered]
In types/lodash the includes method has missing signatures which causes TS to not pick up the correct definition when passing values to it.

See the documentation here: https://lodash.com/docs/4.17.15#includes

[a comment by an automated account is not shown]

**Comment by aj-r** [comment, marked as answer]
Can you add more detail on what signatures are missing? Do you have an example that fails to compile?
